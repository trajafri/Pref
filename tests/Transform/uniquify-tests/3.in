(let ((x 3)) (+ 2 x))
