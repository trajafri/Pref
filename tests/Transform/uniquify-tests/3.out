(let ((x.0 3)) (+ 2 x.0))
